<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoMonitor</class>
 <widget class="QWidget" name="InfoMonitor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(149, 149, 149);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,1,1,1,1,1,1,1,1,1,1">
   <property name="sizeConstraint">
    <enum>QLayout::SetMaximumSize</enum>
   </property>
   <property name="leftMargin">
    <number>11</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>11</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_1" stretch="20,1,20">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QImageLabel" name="labDesc1">
         <property name="text">
          <string>水平定位</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labVal1">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>视觉，gps</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_1">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="10,7,3">
       <item>
        <widget class="QImageLabel" name="labDesc4">
         <property name="text">
          <string>RTK航向</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labVal4">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>19</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labDan4">
         <property name="text">
          <string>度</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="20,1,20">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QImageLabel" name="labDesc2">
         <property name="text">
          <string>垂直定位</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labVal2">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>气压计，视觉，gps</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="10,7,3">
       <item>
        <widget class="QImageLabel" name="labDesc5">
         <property name="text">
          <string>磁1航向</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labVal5">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>19</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labDan5">
         <property name="text">
          <string>度</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_4">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="20,1,20">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QImageLabel" name="labDesc3">
         <property name="text">
          <string>定向</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labVal3">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>磁，rtk</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_5">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="10,7,3">
       <item>
        <widget class="QImageLabel" name="labDesc6">
         <property name="text">
          <string>磁2航向</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labVal6">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>21</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labDan6">
         <property name="text">
          <string>度</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_6">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="20,1,20">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="10,7,3">
       <item>
        <widget class="QImageLabel" name="labDesc7">
         <property name="text">
          <string>GPS高度</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labVal7">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>2m</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labDan7">
         <property name="text">
          <string>米</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_7">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QImageLabel" name="labDesc10">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labVal10">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_8">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="20,1,20">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_14" stretch="10,7,3">
       <item>
        <widget class="QImageLabel" name="labDesc8">
         <property name="text">
          <string>VDop</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labVal8">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>3m</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labDan8">
         <property name="text">
          <string>米</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_9">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_15">
       <item>
        <widget class="QImageLabel" name="labDesc11">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labVal11">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_10">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_16" stretch="20,1,20">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_17" stretch="10,7,3">
       <item>
        <widget class="QImageLabel" name="labDesc9">
         <property name="text">
          <string>HDop</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labVal9">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>2m</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labDan9">
         <property name="text">
          <string>米</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_11">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_18">
       <item>
        <widget class="QImageLabel" name="labDesc12">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labVal12">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QImageLabel</class>
   <extends>QLabel</extends>
   <header location="global">./Utils/qimagelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
